src/signing.js

```diff
@@ -15,31 +17,46 @@ export function verify(data, signature, publicKey) {
   return actualMethods.verify(data, signature, publicKey);
 }
 
-
+var useNacl;
```

Comment:
Why have both `useNacl` and `fastSigning`?  They appear to be always the same value.